125-cd-nextgen/src/main/java/io/harness/cdng/servicenow/resources/service/ServiceNowResourceServiceImpl.java

```
@@ -136,8 +138,20 @@ public List<ServiceNowStagingTable> getStagingTableList(IdentifierRef connectorR
   public List<ServiceNowTicketTypeDTO> getTicketTypesV2(IdentifierRef connectorRef, String orgId, String projectId) {
     ServiceNowTaskNGParametersBuilder parametersBuilder =
         ServiceNowTaskNGParameters.builder().action(ServiceNowActionNG.GET_TICKET_TYPES);
-    return obtainServiceNowTaskNGResponse(connectorRef, orgId, projectId, parametersBuilder)
-        .getServiceNowTicketTypeList();
+    List<ServiceNowTicketTypeDTO> serviceNowStandardTicketTypeList =
+        Arrays.stream(ServiceNowTicketTypeNG.values())
+            .map(ticketType -> new ServiceNowTicketTypeDTO(ticketType.name(), ticketType.getDisplayName()))
+            .collect(Collectors.toList());
+    try {
+      return obtainServiceNowTaskNGResponse(connectorRef, orgId, projectId, parametersBuilder)
+          .getServiceNowTicketTypeList();
+    } catch (HintException ex) {
```

Catch all exception instead of just HintException. The log seems helpful.

---

done